src/components/RefinementList/RefinementList.js

```diff
@@ -100,6 +100,14 @@ class RefinementList extends Component {
       return;
     }
 
+    if (
+      originalEvent.target.matches('input[type="radio"]:checked') &&
```

Comment:
According to MDN, [`Element.matches()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches#Browser_compatibility) isn't supported by IE9.

Reply:
Oh, checked too quickly and didn't saw the icon in the compatibility table. I will update the code to also test `msMatchesSelector`.

Reply:
I wasn't able to try the story on IE9 because storybook doesn't work on this browser. It seems like we never used `Element.matches()` but rather `event.target.querySelector()`:

https://github.com/algolia/instantsearch.js/blob/e610946c77088e6bdbb49916caff0b0ebb2a57d9/src/components/RefinementList/RefinementList.js#L118-L122

Reply:
I rewrote the condition to use `event.target.parentNode.querySelector`.